composer.json

```diff
@@ -19,7 +19,7 @@
     "require": {
         "php": "^7.1",
         "symfony/filesystem": "^4.1",
-        "twig/twig": "^2.5",
+        "twig/twig": "^2.7.3",
```

Comment:
There was some bug fix about the spaceless filter specifically on 2.7.3, so I thought we could just use that as the min.